source/dialog/request_vehicle.sqf

```diff
@@ -39,27 +39,27 @@ _spawnPos = [_spawnPos, 20, 50, 3, 0, 1, 0] call BIS_fnc_findSafePos;
 
 
 switch (_index) do {
-	case 0: { _vehic = [_spawnPos, "I_G_Offroad_01_F", offroad_cost] call spawn_vehicle_fn; };
```

Comment:
i find the change of variable name unnecessary. it's ok to make the change but remember that when integrating the change that can add to the possibility of having merge issues